{"source": "exploit-database", "filename": "platforms/multiple/local/38835.py", "cves": ["CVE-2015-5273", "CVE-2015-5287"]}